### DISCUSSION TITLE
make comment syntax resemble coding comments

### DISCUSSION THREAD

**Discussion by jancbeck** [Fevol/obsidian-criticmarkup#22] [Ideas]
fore purely aesthetic reasons I would love comments markup to resemble code comments in many languages i.e. 
`{## ... ##}` or 
`{** ... **}`

**Comment by Fevol** [comment]
This plugin mainly serves to implement the specification as defined by the [Criticmarkup standard](https://fletcher.github.io/MultiMarkdown-6/syntax/critic.html), defining a completely new syntax for comments would ultimately make your notes less compatible with other tools. 

I'm afraid that I won't implement this (though you are free to fork this plugin, and replace all occurrences of `{>>`, `<<}` with your desired syntax in the code.

**Reply by jancbeck** [reply]
I have released a plugin that uses this syntax:

```
===Some text to annotate===<!--My comment-->
```

This maximizes compatibility as it doesn't introduce new syntax.  

https://github.com/jancbeck/obsidian-note-annotations
